### ISSUE TITLE
Deployment with Windows bootstrap node: Upstream added to the DNS server list

### ISSUE THREAD

**Issue by ionutbalutoiu** [dcos/dcos-windows#133]
We don't need the upstream DNS server added to the end of DNS server list. This is from a recent deployment:

```
PS C:\> Get-DnsClientServerAddress

InterfaceAlias               Interface Address ServerAddresses
                             Index     Family
--------------               --------- ------- ---------------
Ethernet 2                           7 IPv4    {198.51.100.1, 198.51.100.2, 198.51.100.3, 168.63.129.16}
Ethernet 2                           7 IPv6    {fd01:d::c633:6401}
dcos-net                            21 IPv4    {198.51.100.1, 198.51.100.2, 198.51.100.3}
dcos-net                            21 IPv6    {fd01:d::c633:6401, fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff...
Loopback Pseudo-Interface 1          1 IPv4    {198.51.100.1, 198.51.100.2, 198.51.100.3}
Loopback Pseudo-Interface 1          1 IPv6    {fd01:d::c633:6401, fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff...
vEthernet (nat)                     13 IPv4    {198.51.100.1, 198.51.100.2, 198.51.100.3}
vEthernet (nat)                     13 IPv6    {fd01:d::c633:6401, fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff...
vEthernet (b15310e7d0fced5)         17 IPv4    {198.51.100.1, 198.51.100.2, 198.51.100.3}
vEthernet (b15310e7d0fced5)         17 IPv6    {fd01:d::c633:6401, fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff...
```

The `Ethernet 2` is the main interface and it shouldn't have the upstream DNS (`168.63.129.16`) added to the end of the list.

**Comment by paulcallen** [comment]
I am actually confused. On Linux, dcos-net setup prepends 198.51.100.[1,2,3] to the \etc\hosts file. I do the equivalent by putting whatever dns setting is on an interface I just prepend these three addresses which I would have expected to be the same.
The dcos-net-watchdog.py service in dcos-net sees if it can ping one of the prepended entries and if it fails it kills dcos-net service. At that point it should restart the service.
On Linux there is no changing back to defaults if dcos-net is broken, just restarting it.

**Comment by paulcallen** [comment]
I have just pushed an update to this. Take a look and let me know what you think. If it is now resolved let me know.
The watchdog service now runs.
The resolvconf timer service however I do not believe is running and will require Bruce to look at that. It may be that timers are not yet implemented and will require him to do that. I have validated that the steps in the service do in fact run completely though, so as soon as that works we should be complete.
